modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/WorkflowContainer.tsx

```diff
@@ -51,65 +52,79 @@ export default function WorkflowContainer({
 	}
 
 	return (
-		<ClayForm.Group className={className}>
-			<label htmlFor="workflowLabelInput">
-				{Liferay.Language.get('workflow')}
-			</label>
-
-			{!isRootDescendantNode && (
-				<Text as="p" color="secondary" size={2} weight="light">
+		<>
+			{isRootDescendantNode && (
+				<ClayAlert
+					displayType="info"
+					title={`${Liferay.Language.get('info')}:`}
+				>
 					{Liferay.Language.get(
-						'to-set-a-workflow-go-to-applications-process-builder-configuration'
+						'object-definitions-that-inherit-permission-from-a-root-object-definition-also-follow-the-workflow-assigned-to-it'
 					)}
-				</Text>
+				</ClayAlert>
 			)}
-
-			<div className="lfr-objects__workflow-details-section">
-				<ClayInput
-					className="form-control lfr-objects__workflow-input"
-					disabled
-					id="workflowLabelInput"
-					placeholder={Liferay.Language.get('no-workflow')}
-					readOnly
-					value={workflowLabel ? workflowLabel : initialWorkflowLabel}
-				/>
+			<ClayForm.Group className={className}>
+				<label htmlFor="workflowLabelInput">
+					{Liferay.Language.get('workflow')}
+				</label>
 
 				{!isRootDescendantNode && (
-					<div className="lfr-objects__workflow-buttons">
-						<ClayButtonWithIcon
-							aria-label={Liferay.Language.get('refresh')}
-							className="lfr-objects__workflow-reload-button"
-							data-tooltip-align="top"
-							displayType="secondary"
-							onClick={updateWorkflowLabel}
-							symbol="reload"
-							title={Liferay.Language.get('refresh')}
-						/>
+					<Text as="p" color="secondary" size={2} weight="light">
+						{Liferay.Language.get(
+							'to-set-a-workflow-go-to-applications-process-builder-configuration'
```

Comment:
I noticed that this text is slightly different from the mockup. Was this a necessary change, or have the mockups been updated and are suggesting new text?

Also, if this text should be below the input, why did we decide to add it between the label and the input?

Let me know if I got anything wrong here :sweat_smile:

Reply:
@igor-franca good catch. The story that introduced this code, https://liferay.atlassian.net/browse/LPD-49656, is currently under product review and is not related to the scope of this pull request.

@rebsilva  a change is needed, let's address it in the original story. Please discuss with the team if that story needs to be moved back to development.

Reply:
on this case no changed is needed @carolmariaabb, just to let you know!